src/OptionsPanel/OptionsList.tsx

```diff
@@ -96,6 +96,11 @@ export default function OptionsList({
                                             handleUpdateOptionValue={handleUpdateOptionValue(index)}
                                             handleUpdateOptionChecked={handleUpdateOptionChecked(index, multiple)}
                                             readOnly={readOnly}
+                                            disabled={
+                                                multiple &&
+                                                option.checked &&
+                                                options.filter((option) => option.checked).length === 1
+                                            }
```

Comment:
@glaubersilva nice! this could be convenient but let's make it something we opt-in with as a prop like `disableSoloCheckedOption` or `requireOneOptionIsChecked` - that way we can deviate if need be.

Reply:
Resolved in 0425a082717761a81a81ae7501f97ba21965b86a